src/Wellcome.Dds/Wellcome.Dds.Repositories/Presentation/V2/PresentationConverter.cs

```diff
@@ -295,7 +295,13 @@ private T ConvertManifest<T>(Presi3.Manifest p3Manifest, DdsIdentifier identifie
                     if (anno.Items.FirstOrDefault() is SupplementingDocumentAnnotation {Body: var body} &&
                         body is ExternalResource extBody)
                     {
-                        var pageCount = manifest.Metadata.GetValueByLabel("Number of pages");
+                        var pageMetaPair = body.Metadata?.FirstOrDefault(

```

Comment:
is it worth putting that in an extension method if it's likely to be re-usable?

Reply:
maybe later.